### PR TITLE
Default interpretation of missing elevation

### LINKED ISSUE #87: The interpretation of undefined Z (elevation)
I feel the I-D needs to address undefined elevation. Doing it like https://tools.ietf.org/html/rfc5870#section-3.4.5 seems to be the proper thing.

### PR DESCRIPTION
**Pull request by sgillies**
Closes #87

### CODE REVIEW

**Comment by sgillies** [comment]
Ready for :eyes:

File: middle.mkd [hunk 1]
```diff
@@ -280,20 +280,23 @@ The default reference system for all GeoJSON coordinates SHALL be
...
 Other coordinate reference systems, including ones described by CRS
 objects of the kind defined in [GJ2008] are NOT RECOMMENDED. GeoJSON
 processing software SHALL NOT be expected to have access to coordinate
 reference systems databases.  Applications requiring CRS other than the
 default MUST assume all responsibility for reference system and
-coordinate accuracy. Furthermore, GeoJSON coordinates MUST NOT under any
-circumstances use latitude, longitude order. See Section 8,
-Interoperability Considerations, for guidance in processing GeoJSON
-documents that do contain such a CRS object.
+coordinate accuracy and interpretation of missing elevation values.
```
Comment (tschaub) [inline middle.mkd]: Should this be "reference system, coordinate accuracy, and interpretation of missing elevation values"?

Comment (sgillies) [inline middle.mkd]: Yes, that's better.

**Comment by sgillies** [comment]
Thanks @tschaub.

**Pull request merged**